ui/base/src/main/java/org/cru/godtools/base/ui/activity/BaseActivity.java

```diff
@@ -37,6 +39,7 @@ protected void onCreate(@Nullable final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         mAnalytics = AnalyticsService.getInstance(this);
         mEventBus = EventBus.getDefault();
+        Appsee.start();
```

Comment:
does this have to be started for every activity? or can it be started once at the main entry point?

I would really prefer that we don't include the Appsee dependency into the base-ui module if possible, this will cause it to be included in our instant app as well as our main app.

Reply:
Also, Appsee.start() doesn't take the activity context, so maybe a better place to start it is in the main app Application class.

Reply:
https://www.appsee.com/docs/android/native

In the documentation it says to include it in every activity that is an entry point into the app.

Reply:
I tried it in the Application and it didn't work.  I did move it to MainActivity